vllm/model_executor/models/paligemma.py

```diff
@@ -133,12 +132,10 @@ class PaliGemmaMultiModalProjector(nn.Module):
     def __init__(self, vision_hidden_size: int, projection_dim: int):
         super().__init__()
 
-        self.linear = ColumnParallelLinear(vision_hidden_size,
-                                           projection_dim,
-                                           bias=True)
+        self.linear = nn.Linear(vision_hidden_size, projection_dim, bias=True)
```

Comment:
QQ: Why not `ReplicatedLinear`?

Reply:
We haven't officially supported quantized VLMs so keeping them as `nn.Linear` for now.